src/atom-environment.coffee

```diff
@@ -229,6 +229,10 @@ class AtomEnvironment extends Model
 
     checkPortableHomeWritable()
 
+    # Patch the `process.env` on startup to fix the problem first documented
+    # in #4126
+    process.env = @project.getEnv()
```

Comment:
Personally, I was sticking with just "env" as the main part of the name to make it familiar to Node developers and `process.env`. I added "get" to the front to make it clear it is a function rather than just an attribute (and all of the other public APIs have a verb in the front). I can totally be swayed though.